src/30_Advanced/SeatMap.html

```diff
@@ -129,8 +130,8 @@ <h1>Seat map</h1>
       ~-->
       <template type="amp-mustache">
          <!--~
-          We are using `layout=fill` to allow the seatmap to use all the available space in the page, this is because
-          we don't know the height or the ratio od the seatmap.
+          We are using `layout=fill` to allow the seatmap to use all the available space in its container that we set at 5vh, this is because
```

Comment:
Suggestion: "As we don't know the actual size of the seatmap, we are using `layout=fill` for amp-list and amp-pan-zomm to allow the seatmap to use all the available space in its container, which we set to 50% of the viewport height."

(it's better to start an explanation with the reason why you're doing something)

Reply:
Thanks I have updated it